### PR TITLE
[Refactor](executor)Add workload schedule policy table

### PR DESCRIPTION
**Pull request by wangbo**
## Proposed changes
1 Add workload schedule policy schema table to instead of TVF workload_schedule_policy();
2 drop show workload schedule policy stmt
```

mysql [information_schema]>desc workload_schedule_policy;
+-----------+--------------+------+-------+---------+-------+
| Field     | Type         | Null | Key   | Default | Extra |
+-----------+--------------+------+-------+---------+-------+
| ID        | BIGINT       | Yes  | false | NULL    |       |
| NAME      | VARCHAR(256) | Yes  | false | NULL    |       |
| CONDITION | TEXT         | Yes  | false | NULL    |       |
| ACTION    | TEXT         | Yes  | false | NULL    |       |
| PRIORITY  | INT          | Yes  | false | NULL    |       |
| ENABLED   | BOOLEAN      | Yes  | false | NULL    |       |
| VERSION   | INT          | Yes  | false | NULL    |       |
+-----------+--------------+------+-------+---------+-------+
7 rows in set (0.00 sec)


```

### CODE REVIEW

**Comment by wangbo** [comment]
run buildall

**Comment by wangbo** [comment]
run buildall

File: fe/fe-core/src/main/java/org/apache/doris/analysis/SchemaTableType.java [hunk 1]
```diff
@@ -74,7 +74,11 @@ public enum SchemaTableType {
     SCH_ACTIVE_QUERIES("ACTIVE_QUERIES", "ACTIVE_QUERIES", TSchemaTableType.SCH_ACTIVE_QUERIES),
     SCH_WORKLOAD_GROUPS("WORKLOAD_GROUPS", "WORKLOAD_GROUPS", TSchemaTableType.SCH_WORKLOAD_GROUPS),
     SCHE_USER("user", "user", TSchemaTableType.SCH_USER),
-    SCH_PROCS_PRIV("procs_priv", "procs_priv", TSchemaTableType.SCH_PROCS_PRIV);
+    SCH_PROCS_PRIV("procs_priv", "procs_priv", TSchemaTableType.SCH_PROCS_PRIV),
+
+    SCH_WORKLOAD_SCHEDULE_POLICY("WORKLOAD_SCHEDULE_POLICY", "WORKLOAD_SCHEDULE_POLICY",
+            TSchemaTableType.SCH_WORKLOAD_SCHEDULE_POLICY);
+
```
Comment (yiguolei) [inline fe/fe-core/src/main/java/org/apache/doris/analysis/SchemaTableType.java:81]: And also delete related show proc code, we will not maintain show proc in the future.

**Comment by wangbo** [comment]
run buildall

**Comment by wangbo** [comment]
run buildall

**Review by yiguolei — APPROVED** [review]

**Review by xinyiZzz — APPROVED** [review]
LGTM

**Pull request merged**